share/completions/iwctl.fish

```diff
@@ -0,0 +1,61 @@
+set adhoc '(iwctl ad-hoc list | tail -n -2 | awk \'{print $1}\')'
+set ap '(iwctl ap list | tail -n -2 | awk \'{print $1}\')'
```

Comment:
This prints "No" for me. Presumably it should be empty instead.

Unfortunately `iwctl` provides no useful exit code (it still exits 0 even if no devices in ap mode are there), so you might have to simply ignore "No"